LabExT/ExperimentManager.py

```diff
@@ -204,6 +204,10 @@ def setup_runner(self):
         self.exp.import_measurement_classes()
         # then we load all Instruments
         self.instrument_api.load_all_instruments()
+        # then we load all Stages and Settings
```

Comment:
the loaded stages should then also be added to the plugins loaded logging (done in the `__init__` method here) so it can be seen in the main window logging. Right now they are listed under lvcards

Reply:
Added it, now here: 
https://github.com/LabExT/LabExT/pull/147/files#diff-01b358552e5d1a58354dc2dde4fe850bbf15cb68e78b2f69c8fc18e6b92099e8R151